src/jdk.incubator.concurrent/share/classes/jdk/incubator/concurrent/StructuredTaskScope.java

```diff
@@ -222,10 +222,42 @@
  *   scope "B" is the parent of the nested task scope "C".
  * </ul>
  *
- * <p> The tree structure supports confinement checks. The phrase "threads contained in
- * the task scope" in method descriptions means threads started in the task scope or
- * descendant scopes. {@code StructuredTaskScope} does not define APIs that exposes the
- * tree structure at this time.
+ * <p> The tree structure supports:
+ * <ul>
+ *   <li> Inheritance of {@linkplain ScopedValue scoped values} across threads.
+ *   <li> Confinement checks. The phrase "threads contained in the task scope" in method
+ *   descriptions means threads started in the task scope or descendant scopes.
```

Comment:
sadly, the term "descendant scopes" is not defined elsewhere in this javadoc

Reply:
Good poin,t, we had more setup in previous iterations.
@theRealAph I'll adjust this in the loom repo, can't do it here.